src/libraries/System.Security.Cryptography.Algorithms/tests/HKDFTests.cs

```diff
@@ -530,6 +566,17 @@ public void Rfc5869OkmMaxSizePotentiallyOverflowingValue()
                     () => HKDF.Expand(HashAlgorithmName.SHA1, prk, okm, Array.Empty<byte>()));
             }
 
+            [Fact]
+            public void Rfc5869ExpandOutputLengthZero()
+            {
+                byte[] prk = new byte[20];
+                byte[] okm = new byte[0];
+
+                AssertExtensions.Throws<ArgumentOutOfRangeException>(
```

Comment:
Won't this test and the next one fail because they're throwing ArgumentException now?